### ISSUE TITLE
[Premium] what is the roadmap for release for Azure Functions premium plan?

### ISSUE THREAD

**Issue by PingPongSet** [Azure/Azure-Functions#1235]
What is the roadmap for release for Azure Functions premium plan?

**Comment by waltersjack** [comment]
Yes, please let us know!  So far is seems far superior to the current premium plans.

**Comment by alexkarcher-msft** [comment]
We're currently working to get the Premium plan production ready. We have no official target, but stay tuned to our blog and twitter feed for more updates in the coming months.

Is there any specific functionality you're excited to get access to in production?

**Comment by waltersjack** [comment]
This is just from what I have observed in the past week working with it.
- BIG PLUS -> pre warmed instances
- Better reliability when connecting to storage accounts and cosmos table
-- I was having some issues with cold starts connecting to azure table storage, but it was still much more reliable that with the current P1v2 plan - changed over to Cosmos DB Table and all the issues vanished
- More reliable request rates (on our app was averaging about 50 ms per request with around 3.5k requests per second and 1k users)

**Comment by FrankGagnon** [comment]
Hi @alexkarcher-msft,

Are there any updates on the roadmap of the premium functions? Both the ability to pre-warm instances and the ability to connect to a VNet would make Azure Functions a valid option for our application.

On a side note, are there any plans to support a wider range of SKUs? 

Thanks

**Comment by raorugan** [comment]
We are eager to learn about you experience, any challenges that you are facing and gather feedback to enhance  experience with Azure Functions. As we are in the planning phase your feedback is invaluable and can significantly influence our product roadmap, increasing the likelihood of incorporating their feature asks and suggestions into the product.
Hi @FrankGagnon , love to learn more about range of SKU requirement we are currently thinking about this capability. Your inputs will be valuable.
We would greatly appreciate it if you could take a few moments to fill out a [survey](https://microsoft.qualtrics.com/jfe/form/SV_1QPVK42KMzFw5G6) to help us improve our services.
